### ISSUE TITLE
arria-pressed attribute is applied to non toggleable buttons

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#2941]
**Describe the bug**
Tested with NVDA in Chrome, default [Kendo Button](https://www.telerik.com/kendo-angular-ui/components/buttons/button/) is read as '_toggle button_'. 
The _aria-pressed_ attributed is applied even if the buttons are not toggleable.

**To Reproduce**
Here is a screencast:
https://screencast-o-matic.com/watch/cY1QXOC6Pd

https://stackblitz.com/edit/angular-x9ujpj?file=app%2Fapp.component.ts

**Expected behavior**
All buttons should have _role='buton'_ and _aria-pressed_ attribute only when the buttons are [toggleable](https://www.telerik.com/kendo-angular-ui/components/buttons/button/toggleable/).

Reported in private support thread: **1470856**

**Comment by siyka-marinova** [comment]
Fixed in the latest [development build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of `kendo-angular-buttons`.

**Comment by preslavapetrova1** [comment]
Fixed in the latest official version `kendo-angular-buttons v5.4.4`
